base/linalg/lapack.jl

```diff
@@ -775,7 +970,39 @@ for (gesvx, elty, relty) in
             X, rcond, ferr, berr, rpgf
         end
     end
- end
+end
+
+"""
+    gesvx!(fact, trans, A, AF, ipiv, equed, R, C, B) -> X, equed, R, C, B, rcond, ferr, berr, work
+
+Solves the linear equation `A * X = B` (`trans = N`), `A.' * X =B`
+(`trans = T`), or `A' * X = B` (`trans = C`) using the `LU` factorization
+of `A`. `fact` may be `E`, in which case `A` will be equilibrated and copied
+to `AF`; `F`, in which case `AF` and `ipiv` from a previous factorization
+are inputs; or `N`, in which case `A` will be copied to `AF` and then
+factored. If `fact = F`, `equed` may be `N`, meaning `A` has not been
+equilibrated; `R`, meaning `A` was multiplied by `diagm(R)`; `C`, meaning `A`
```

Comment:
there's a distinction between R and C of left-multiplying vs right-multiplying by the scaling matrices, right?